src/TabBarItem.tsx

```diff
@@ -63,21 +66,24 @@ export default class TabBarItem<T extends Route> extends React.Component<
     }
   );
 
-  private getInactiveOpacity = memoize((position, routes, tabIndex) => {
```

Comment:
Looks like this method didn't change. Can we revert this part of the diff to what it was?

Reply:
I've reverted this change, although it added types to the main function params